zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/CamundaExporter.java

```diff
@@ -66,18 +66,19 @@ public class CamundaExporter implements Exporter {
   private final ExporterResourceProvider provider;
   private CamundaExporterMetrics metrics;
   private BackgroundTaskManager taskManager;
-  private final ExporterMetadata metadata;
+  private ExporterMetadata metadata;
   private boolean importersCompleted = false;
   private SearchEngineClient searchEngineClient;
   private int partitionId;
 
   public CamundaExporter() {
-    this(new DefaultExporterResourceProvider());
+    // the metadata will be initialized on open
+    this(new DefaultExporterResourceProvider(), null);
   }
 
   @VisibleForTesting
   public CamundaExporter(final ExporterResourceProvider provider) {
-    this(provider, new ExporterMetadata());
+    this(provider, null);
```

Comment:
🔧 Or could we also do it here: `new ExporterMetadata(clientAdapter.objectMapper());`